src/extractor.rs

```diff
@@ -690,6 +690,12 @@ macro_rules! tuple_from_req ({$fut_type:ident, $(($n:tt, $T:ident)),+} => {
     }
 });
 
+impl<S> FromRequest<S> for () {
+    type Config = ();
+    type Result = Self;
+    fn from_request(_req: &HttpRequest<S>, _cfg: &Self::Config) {}
```

Comment:
It is unnecessary, but I kinda feel we better follow the original trait definition and add `-> Self::Result`
If you don't think so, you can ignore 😄

Reply:
Done.